### ISSUE TITLE
metalnx upload hangs when uploading a folder / directory

### ISSUE THREAD

**Issue by anderbubble** [irods-contrib/metalnx-web#116]
Metalnx accepts a request to upload a folder/directory; but then it hangs apparently indefinitely with an arbitrary low percentage complete.

I'm hoping Metalnx is meant to be able to handle directory uploads as a single operation; but, if not, it should report this lack up support when the operation is requested, and fail immediately.

**Comment by michael-conway** [comment]
are you doing this via the upload dialog? That is, select upload, and then
pick a directory and it attempts to do so? What browser/os are you on? On
Mac/Chrome it only allows you to select a file or multiple files...It may
be at this level we need to look at it.

**Comment by anderbubble** [comment]
I think I was using the drag-and-drop support in the upload dialog with Safari. I'll try it with another browser, too.

**Comment by michael-conway** [comment]
Ok and yes we should add some code on the back end to catch when a folder
upload is happening b/c that's not actually supported, Mutli file is though.

**Comment by anderbubble** [comment]
Any chance it _could_ be supported? :)

**Comment by michael-conway** [comment]
Indeed, that would be nice, it might be a thing to split into a bug and a
feature
